index.js

```diff
@@ -0,0 +1,82 @@
+const { exec } = require("child_process");
+const fs = require('fs');
+/**
+ * This function leverages the is-my-node-vulnerable package
+ * (https://github.com/RafaelGSS/is-my-node-vulnerable/blob/main/index.js) and
+ * extends it to be able to suppress / expire suppressions for pipeline use.
+ */
+function scanNode(){
+    exec('npx is-my-node-vulnerable@1.4.1',(error, stdout, stderr) => {
```

Comment:
Looks like `is-my-node-vulnerable` has an [API](https://github.com/RafaelGSS/is-my-node-vulnerable/blob/main/README.md#api). Have you tried running this scan as
```
const { isNodeVulnerable } = require('is-my-node-vulnerable')

isNodeVulnerable(process.version)
```

instead of using `npx`? Just curious whether it would be more efficient/easier to do.

Reply:
I've tried that, the response returns only a boolean telling you if node is vulnerable or not, and doesn't provide any of the output that would be useful to us